amenity-points.mss

```diff
@@ -218,6 +218,34 @@
     marker-clip: false;
   }
 
+  [feature = 'leisure_pitch'][sport = 'baseball'][zoom >= 17] {
```

Comment:
You can combine all the sports under one `[feature = 'leisure_pitch'] rather than repeating this 4 times, eg:
```
  [feature = 'leisure_pitch'][zoom >= 17] { 
    marker-fill: @leisure-green;
    marker-placement: interior;
    marker-clip: false;
    [sport = 'baseball'][zoom >= 17] {
      marker-file: url('symbols/sport/baseball.svg');
    }
    [sport = 'basketball'][zoom >= 17] {
      marker-file: url('symbols/sport/basketball.svg');
... etc.
```
And then add a SQL query in project.mml , I think something like:
```
     CASE WHEN tags->'sport' IN ('baseball', 'basketball', 'soccer', 'tennis') then sport ELSE NULL as sport,
```
instead of `tags->'sport' as sport,`

Reply:
I might give it a try. If not, maybe someone else can do it later after its implemented. I was under the impression that its better not to do an SQL query if you don't have to though. Since its what slows down rendering. There's enough `CASE WHEN tags->` instances already in my opinion.